GliaWidgets/Coordinator/RootCoordinator.Environment.swift

```diff
@@ -0,0 +1,23 @@
+extension RootCoordinator {
+    struct Environment {
+        var chatStorage: Glia.Environment.ChatStorage
+        var fetchFile: CoreSdkClient.FetchFile
+        var sendSelectedOptionValue: CoreSdkClient.SendSelectedOptionValue
+        var uploadFileToEngagement: CoreSdkClient.UploadFileToEngagement
+        var audioSession: Glia.Environment.AudioSession
+        var uuid: () -> UUID
+        var fileManager: FoundationBased.FileManager
+        var data: FoundationBased.Data
```

Comment:
This one also has a very non-descriptive name

Reply:
I added explanation [here](https://github.com/salemove/ios-sdk-widgets/pull/346#discussion_r933312932).